config/m4/sysdep.m4

```diff
@@ -106,13 +106,16 @@ AS_IF([test "x$with_valgrind" == xno],
 #
 AC_ARG_ENABLE([numa],
     AC_HELP_STRING([--disable-numa], [Disable NUMA support]),
-    [AC_DEFINE([UCT_MD_DISABLE_NUMA], 1, [Undefine to enable NUMA support])],
+    [
+        AC_MSG_NOTICE([NUMA support is disabled])
+        AC_DEFINE([UCT_MD_DISABLE_NUMA], 1, [Undefine to enable NUMA support])
+    ],
     [
         AC_CHECK_HEADERS([numa.h numaif.h], [],
-                         [AC_MSG_ERROR([NUMA headers not found])])
+                         [AC_MSG_ERROR([NUMA headers not found. Please reconfigure with --disable-numa])])
```

Comment:
The error output will be:
```
checking numa.h usability... no
checking numa.h presence... no
checking for numa.h... no
configure: error: NUMA headers not found. Please reconfigure with --disable-numa.  Warning: this may have negative impact on library performance. It is better to install libnuma-devel rpm
```